src/lib389/lib389/cli_idm/account.py

```diff
@@ -118,6 +119,52 @@ def subtree_status(inst, basedn, log, args):
         _print_entry_status(status, entry.dn, log)
 
 
+def bulk_update(inst, basedn, log, args):
```

Comment:
Could you please also add a simple test? Something similar to this:

https://github.com/389ds/389-ds-base/blob/7efbeb3b75a5f879dec87a90ff925b436216c4e6/dirsrvtests/tests/suites/clu/dsidm_services_test.py#L206-L239